docs/docs/gcp-bigquery.md

```diff
@@ -0,0 +1,257 @@
+---
```

Comment:
@eder001 I think this might contain too much marketing, rather then specific instructions.  Can you coordinate with the BigLake/BigQuery team to revise internally and then open reopen the PR.